source/_components/binary_sensor.envisalink.markdown

```diff
@@ -14,5 +14,26 @@ ha_iot_class: "Local Push"
 ---
 
 The `envisalink` platform allows you to get data from your [Envisalink](http://www.eyezon.com/) binary sensors from within Home Assistant.
-  
+
 The requirement is that you have setup your [Envisalink hub](/components/envisalink/).
+
+#### Sensor Types
```

Comment:
No, we can't remove the Envisalink binary sensor docs because we are linking from the configuration check to the docs.